syllabus/2-Core_and_Basics/0-light-client-tx-slides.md

```diff
@@ -0,0 +1,250 @@
+
+All of this can again be independently verified by the nodes on the network, including light clients.
+
+---
+
+## Following the Validator Set
+
+Verifying the Justification assumes the light client knows the current validator set. Since it does not actually execute the blocks, it cannot simply query for that information.
+
+Instead, this information is constantly updated in the block digest along with the other consensus critical data.
+
+---
+
+## Updating the Validator Set
+
+The genesis block defines the initial state of the blockchain, including the initial validator set.
+
+From there, based on the state transition function of the blockchain, a new validator set might be queued for some future block.
+
+This will be pre-announced in the digest, and signed by the current validator set, showing that a new validator set will be active in the future.
+
+With this announcement included in the digest, and backed by the GRANDPA Justification, light clients can always know who the active validators are, even as they change.
+
+---
+
+## Light Clients Verify Everything but the STF
+
+As you can see, Light Clients are able to remain trustless within the blockchain ecosystem because they are able to verify all of the block headers of the canonical chain.
+
+With this, it becomes simple to verify:
+
+- a transaction has been included in the chain, from the extrinsics root.
+- the state of the chain, from the state root.
+
+And remember, the State Transition Function (Wasm) itself is stored on chain, but the light client is not expected to execute it.
+
+---
+
+# Part 1: Light Client Wants to Create a Transaction
+
+---
+
+So we have seen so far, in general, how a light client would trustlessly follow the canonical version of the blockchain.
+
+But let's assume the light client wants to interact with the chain. Something simple like a balance transfer from Alice to Bob.
+
+How would it do that?
+
+---
+
+## Fetching the Metadata
+
+The first thing the light client will do is fetch the up to date metadata of the chain.
+
+Remember that in the Polkadot ecosystem, chains can constantly upgrade and update their state transition function, changing the functionality of the chain and even what extrinsics are available.
+
+---
+
+## State Transition Function is Self Describing
+
+To get the metadata itself, we must query the Runtime via the State Transition Function Code.
+
+The Runtime exposes an API: `state_getMetadata`, which will return a SCALE encoded blob with all the data you need to know about the blockchain.
+
+---
+
+## Merkle Tree
+
+TODO
+
+---
+
+## Merkle Proof
+
+TODO
+
+---
+
+## SCALE
+
+TODO
+
+---
+
+## FRAME Metadata Format
+
+- Versioned, and occasionally changes over time.
+- Defined by: https://github.com/paritytech/frame-metadata
+- Automatically generated if you use FRAME macros.
+- Output can be turned into JSON, which is easiest for human readability:
+  - https://dev.papi.how/metadata/json
```

Comment:
@shawntabrizi Note that we'll have already gone into SCALE, Metadata and Runtime (and JSON-RPC). We're currently updating all of those slides by adding more hands-on and less text ([current diff](https://github.com/Polkadot-Blockchain-Academy/pba-content/compare/main...polkadot-api:pba-content:fundamentals#diff-70bded7b7c2181ec6358df57a1e24666bfdec2a22d5727b1e00edd43db5abdad)). Unless you want to mention something specific that's important or works as an introduction to something else, it shouldn't be needed to repeat it.

I think the idea in terms of ordering is:

1. SCALE
2. Runtime
3. Metadata
4. JSON-RPC
5. Light Clients (and putting everything together)

Reply:
i dont think it harms to make the slides themselves more independent and to quickly retouch on existing topics for emphasis

i dont plan to expand deeply into anything that has already been covered